# Local change to Circle Step51.1 (c776b3c6): FatFs with f_mkfs, so that the
# installer (gpu/install) can put a FAT32 file system on a blank SD card. Apply with:
#   cd circle && git apply ../patches/circle-fatfs-mkfs.patch

diff --git a/addon/fatfs/ffconf.h b/addon/fatfs/ffconf.h
index 9011a9a8..bafda64f 100644
--- a/addon/fatfs/ffconf.h
+++ b/addon/fatfs/ffconf.h
@@ -30,7 +30,7 @@
 /  f_findnext(). (0:Disable, 1:Enable 2:Enable with matching altname[] too) */
 
 
-#define FF_USE_MKFS		0
+#define FF_USE_MKFS		1
 /* This option switches f_mkfs(). (0:Disable or 1:Enable) */
 
 
